roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -50,7 +50,10 @@ Disputes may occur against blocks that have happened in the session prior to the
 In this case, the prior validator set is responsible for handling the dispute and to do so with their keys from the last session.
 This means that validator duty actually extends 1 session beyond leaving the validator set.
 
-Wait for a 2/3 majority either way, by counting incoming votes. Or timeout.
+Wait for a 2/3 majority either way, by counting incoming votes. If there is no super majority reached within a timeout, this indicates a bipartition of the
+network with its validators, this should never happen given the timeout is set
+generously under our assumptions, hence partial slashing of all valditors
+is ok.
```

Comment:
We don't slash anybody if we cannot resolve the dispute, but we do consider the block invalid, and should notify humans so they could create a motion for manual slashing.